### ISSUE TITLE
MANIFEST.UNPLAYABLE_PERIOD error on HLS manifest

### ISSUE THREAD

**Issue by PidginEnemy** [google/shaka-player#885]
**Have you read the [FAQ](https://goo.gl/JE1Sy5) and checked for duplicate issues**:
yes
**What version of Shaka Player are you using**:
2.1.3
**Can you reproduce the issue with our latest release version**:
yes
**Can you reproduce the issue with the latest code from `master`**:
i'm not tried 'master'
**Are you using the demo app or your own custom app**:
demo
**If custom app, can you reproduce the issue using our demo app**:
yes
**What browser and OS are you using**:
Firefox(latest), Chrome(latest), Opera(latest)
**What are the manifest and license server URIs**:
*(you can send the URIs to <shaka-player-issues@google.com> instead, but please use GitHub and the template for the rest)*
https://media-ci.foxford.ru/dist/hls-issue/issue.master.m3u8

**What did you do?**
Paste my manifest in demo example

**What did you expect to happen?**
Playing video

**What actually happened?**
Error - MANIFEST.UNPLAYABLE_PERIOD (4011)

My HLS manifest not working in Shaka player (latest version).

I tested this in demo application

select the "custom asset"
paste this https://media-ci.foxford.ru/dist/hls-issue/issue.master.m3u8 in "Custom manifest" field
This is my solution

Shaka player show error (MANIFEST.UNPLAYABLE_PERIOD, code 4011) - http://prntscr.com/fhgcua

I hope you help me with this problem
Thanks,
Evgeny

**Comment by joeyparrish** [comment]
We have an entry about this in the FAQ:

> **Q:** My HLS manifest doesn't load.
> 
> **A:** If your HLS manifest describes MPEG2-TS content, the only browsers
capable of playing it are Edge, Chromecast and Safari. You will get an
`UNPLAYABLE_PERIOD` error on other browsers due to their lack of TS support.
> 
> We also were not able to make it work on Safari yet due to a bug in their
MediaSource implementation (#743).
> 
> Please file a issue if your TS content isn't playing in Chromecast or Edge
and your MP4 content - on any browser.

We also recognize that this error message is confusing, and we have proposed a new error code in #868 that would make the problem more clear: `CONTENT_NOT_SUPPORTED_BY_BROWSER`.

Does this answer your question?

**Comment by joeyparrish** [comment]
After some internal discussion, we have decided that this is consistently surprising to developers, and that we can do better.  I filed #887 for Shaka Player to convert TS to fMP4 so that we could play TS in all browsers.  I do not have a date when this would be ready, and we still have lots of research to do.

For now, this is working as intended, but we will look into TS transmuxing for the future.  I apologize for the inconvenience.
